src/EventSourcingTests/Projections/Flattened/ValuesSubtracted.cs

```diff
@@ -0,0 +1,10 @@
+namespace EventSourcingTests.Projections.Flattened;
+
+public class ValuesSubtracted
+{
```

Comment:
**SUGGESTION:** I think that'd be good to add also at least fields with `DateTimeOffset` and `Guid`.

Reply:
As a test? Should be able to handle any type that Weasel + Npgsql handles as is.